src/main/resources/application.properties

```diff
@@ -69,3 +69,5 @@ com.ampnet.identityservice.veriff.private-key=
 com.ampnet.identityservice.veriff.api-key=
 
 com.ampnet.identityservice.test.enabled-test-kyc=true
+
+com.ampnet.identityservice.provider.blockchain-api=https://eth-goerli.alchemyapi.io/v2/Geqo-GtNKkGpfFbcCR2-67gPzI7wpTRd
```

Comment:
When you have a default value set in the Kotlin properties file, you do need to set it again in the `application.properties` - you can omit it.

Reply:
ok, I will remove it.